internal/db/schema/migrations/postgres/16/04_plugin_host.up.sql

```diff
@@ -0,0 +1,157 @@
+begin;
+
+/*
+                             ┌──────────────────┐
+                             │   host_plugin    │
+                             ├──────────────────┤
+                             │public_id (pk)    │
+                             │...               │
+                             └──────────────────┘
+                                       ┼
+                                       ┼
+                                       ○
+                                      ╱│╲
+                            ┌─────────────────────┐
+                            │ plugin_host_catalog │
+                            ├─────────────────────┤      ┌───────────────────────────┐
+    ┌────────────────┐      │public_id (pk)       │      │plugin_host_catalog_secret │
+    │host_catalog    │      │plugin_id (fk)       │      ├───────────────────────────┤
+    ├────────────────┤      │scope_id (fk)        │      │host_catalog_id (pk, fk)   │
+    │public_id       │┼┼──○┼│name                 │┼┼──○┼│secret                     │
+    │scope_id        │      │description          │      │key_id (fk)                │
+    └────────────────┘      │attributes           │      └───────────────────────────┘
+             ┼              └─────────────────────┘
+             ┼                         ┼
+             │                         ┼
+             │                         ○
+             ○                        ╱│╲
+            ╱│╲             ┌────────────────────┐
+    ┌────────────────┐      │  plugin_host_set   │
+    │    host_set    │      ├────────────────────┤
+    ├────────────────┤      │public_id (pk)      │
+    │public_id       │      │host_catalog_id (fk)│
+    │host_catalog_id │┼┼──○┼│name                │
+    └────────────────┘      │description         │
+                            │attributes          │
+                            └────────────────────┘
+
+*/
+
+  create table plugin_host_catalog (
+    public_id wt_public_id primary key,
```

Comment:
Does the prefix of the `public_id` have to match the `id_prefix` of the `plugin_id`?